README.md

```diff
@@ -1,57 +1,63 @@
 # Guide
 
-A website to teach Wagtail CMS to content editors, moderators and administrators.
+Guide is a website to help content editors, moderators, administrators, and other users learn how to user the Wagtail content management system (CMS).
 
-Wagtail guide brings you:
-- Tutorials, 
-- How-to, 
-- Reference 
-- Background information
+The Wagtail guide will ultimately include:
+- Tutorials 
+- How-to articles
+- Reference materials
+- Walkthroughs and visual learning materials
```

Comment:
This list was about the sections/modes we adopt. Isn't it better to stay close to the original writing modes?

- Tutorials 
- How-to
- Explanation
- Reference

We could explain a bit more: https://documentation.divio.com/